packages/world/script/TestScript.s.sol

```diff
@@ -42,31 +42,31 @@ contract TestScript is Script {
     EntityId playerEntityId = Player.get(0xE0ae70caBb529336e25FA7a1f036b77ad0089d2a);
     require(playerEntityId.exists(), "Player entity not found");
     // testRemoveFromInventoryCount(playerEntityId, 162, 10);
-    testAddToInventoryCount(playerEntityId, PlayerObjectID, BedrockObjectID, 4);
-    testAddToInventoryCount(playerEntityId, PlayerObjectID, ChipObjectID, 10);
-    testAddToInventoryCount(playerEntityId, PlayerObjectID, ChipBatteryObjectID, 99);
-    testAddToInventoryCount(playerEntityId, PlayerObjectID, ForceFieldObjectID, 2);
-    testAddToInventoryCount(playerEntityId, PlayerObjectID, TextSignObjectID, 15);
-    testAddToInventoryCount(playerEntityId, PlayerObjectID, SmartTextSignObjectID, 15);
-    testAddToInventoryCount(playerEntityId, PlayerObjectID, PipeObjectID, 99);
-    testAddToInventoryCount(playerEntityId, PlayerObjectID, PowerStoneObjectID, 1);
-    testAddToInventoryCount(playerEntityId, PlayerObjectID, OakLogObjectID, 99);
-    testAddToInventoryCount(playerEntityId, PlayerObjectID, ChestObjectID, 1);
-    testAddToInventoryCount(playerEntityId, PlayerObjectID, SmartChestObjectID, 4);
-    testAddToInventoryCount(playerEntityId, PlayerObjectID, BlueGlassObjectID, 99);
-    testAddToInventoryCount(playerEntityId, PlayerObjectID, CoalOreObjectID, 99);
-    testAddToInventoryCount(playerEntityId, PlayerObjectID, SilverBarObjectID, 99);
-    testAddToInventoryCount(playerEntityId, PlayerObjectID, SandObjectID, 99);
-    testAddToInventoryCount(playerEntityId, PlayerObjectID, StoneObjectID, 99);
+    // testAddToInventoryCount(playerEntityId, PlayerObjectID, BedrockObjectID, 4);
```

Comment:
I wonder whats a good replacement for this?

Ie when testing in the client you often want to spawn in a bunch of resources, eg force field or pick axe

Reply:
Ah damn not sure, let's discuss